docs/api-reference/effects/glfx-filters.md

```diff
@@ -0,0 +1,343 @@
+# glfx Filters
+
+
+Image effects packaged as reusable shader modules based on the [glfx library](http://evanw.github.io/glfx.js/).
+
+
+## Attribution / License
+
+This is mostly a repackaging of shader code from [Evan Wallace](https://github.com/evanw/glfx.js)'s wonderful glfx library.
+
+The code and documentation is included here under MIT license.
+
+
+## Filters
+
+
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td align="center">
+        <img height=340 src="https://raw.githubusercontent.com/uber-common/deck.gl-data/master/images/samples/glfx/mountain.jpg" />
+        <p><i>Original Image</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+### brightnessContrast
+
+Provides additive brightness and multiplicative contrast control.
+
+* `brightness` -1 to 1 (-1 is solid black, 0 is no change, and 1 is solid white)
+* `contrast`   -1 to 1 (-1 is solid gray, 0 is no change, and 1 is maximum contrast)
+
+<table style="border: 0;" align="center">
```

Comment:
No need to use HTML if you don't need any custom layout.

`![brightnessContrast](https://raw.githubusercontent.com/uber-common/deck.gl-data/master/images/samples/glfx/results/brightness.jpg)`

Reply:
with HTML style it is easier to control the image size and align the `Brightness / Contrast Effect` text with the image.